confirm-deposit.md

```diff
@@ -0,0 +1,120 @@
+# Confirm Deposits to the Exchange
+
+Deposits are made in several stages. 
+After the user deposits funds to their assigned reception wallet (step 1 in [Detecting Deposits](https://docs.flare.network/exchange/architecture/#detecting-deposits)), the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange) monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a [blockchain](https://docs.flare.network/tech/glossary/), multiple [validators](https://docs.flare.network/tech/validators/) examine newly created transactions and decide on whether they are valid and which [block](https://docs.flare.network/tech/glossary/) to add to the chain next. 
+It takes time for all the validators to agree. 
+Until they reach agreement, validators may add or revert transactions to a block. 
+After the validation process is complete, reversing a transaction is very unlikely. 
+To avoid the risk of confirming a block that could be reverted, confirm deposits several blocks back (for example, five blocks back as in the code below). 
+Older blocks are less likely to be reverted.
+
+Newly submitted deposits to the receiving address are labeled "pending."
+After five more blocks have been created, valid deposits are labeled "confirmed."
```

Comment:
```suggestion
After five more blocks have been created, valid deposits are then labeled "confirmed."
```

Again, I like creating a story rather than having disconnected sentences. Up for debate.

Reply:
I was thinking that "after" clarifies the timeline, but developers' first language may be have them thinking it the "if-then" pattern. So, agreed. 
In general, I try to avoid redundancy, but sometimes it is good for emphasis and helps the reader. Notice I use the phrase "your own" on line 65...that's to help them hear it more clearly.